src/index.js

```diff
@@ -52,3 +53,142 @@ const init = () => {
 };
 
 init();
+// Rework
+class customInput{
+  constructor(title){
+    this.cont = document.createElement('div');// wrapper for input and buttons
+    this.name = document.createElement('h2');// title for input
+    this.input = document.createElement('input');// input element
+    this.tags = document.createElement('ul');// list for tags
+    this.name.textContent = title;
+    this.name.className = 'title-level-2';
+    this.input.placeholder = 'Type tag';
+    this.input.className = 'tags-input';
+    this.tags.className = 'tags-list';
+    this.cont.className = 'cont';
+    this.tagsArr = [];
+    this.tagsEnable = true;
+    this.cont.append(this.input);
+    main.append(this.name, this.cont, this.tags);
+    this.hintElem = new customHint(this);
+    main.append(this.hintElem);
+
+    addEvents(this);
+  }
+  defaultTags(){
+    const tagsList = ['hi', 'hello', 'bye'];
+
+    this.tagsArr = [...tagsList];
+    for(let value of this.tagsArr) this.tags.append(createTagElement(value));
+  }
+  addButton(name){
+    let btn = document.createElement('button');
```

Comment:
you didn't change the value of this let variables, so it must be const declaration
👎